modules/amp-core/pages/07-data-bindings.adoc

```diff
@@ -1,9 +1,102 @@
+
 = Using data bindings
 
+The player continues to support very advanced and flexible parameterization. Substitutions in many configuration elements (usually for ads and analytics) continue to use the the syntax `#{variable}` or `${variable}`.
+
+The following data values are accessible through player data bindings
+
+* Player provided
+** Media
+** Ads
+** l10n
+* Customer provided params [Deprecated]
+* Wide-open JS / DOM execution
+** DOM Variables
+** JS Functions
+
+=== Player provided data
+==== Media
+This is a media object representation, it exposes the values of the current playing media or mrss feed in the video player. See the list of https://docs.player.akamai.com/static/amp-web-sdk/akamai.amp.Media.html[Media] members. 
+Example
+
+[source,javascript]
+----
+config = {
+    plugins: {
+        analyticsPlugin: {
+            title: "#{media.title}",
+            eventName: "#{media.title}",
+            isfullepisode:"#{media.duration / 3600 >= 1 ? 'Y': 'N'}",
+        }
+    }
+----
+
+==== Ads
+Player exposes contextual ad data during ad playback, ad members can be accessed through player data bindings but keep in mind that the ad should be active prior accessing its members. 
+Example.
+
+[source,javascript]
+----
+config = {
+    plugins: {
+        analyticsPlugin: {
+            adTitle: "${ads.currentAd.id}", // 123456 
+            adPosition: "${ads.currentAd.type}", // preroll
+        }
+    }
+----
+
+see  
+https://docs.player.akamai.com/static/amp-web-sdk/akamai.amp.Ad.html[Ad contextual data]
+
+==== Player
+Player provided data such as `<video>` information.
+
 [source,javascript]
 ----
-"${player.mode} Player"
+
+config = {
+    plugins: {
+        analyticsPlugin: {
+            heigth: "${player.videoHeight}", // Current video Heigth 
+            width: "${ads.currentAd.type}", // Current video Width
```

Comment:
@mchacong 
this example seems incorrect,

isn't width: "${player.videoWidth}" 
???

Reply:
yes, fixed